### ISSUE TITLE
"No deserializable introspection present for type:" When adding "io.micronaut.mqtt:micronaut-mqttv5" dependency

### ISSUE THREAD

**Issue by breucode** [micronaut-projects/micronaut-mqtt#332]
### Expected Behavior

micronaut-serde should not be used at all, since I already have jackson-databind in use

This is the same problem as https://github.com/micronaut-projects/micronaut-problem-json/issues/257

It can be fixed by adding the following to the `build.gradle.kts`

```
configurations.all {
  exclude(group = "io.micronaut.serde")
}
```

### Actual Behaviour

The error "No deserializable introspection present for type:" is thrown as soon as anything in the project tries to serialize or deserialize json.

### Version

4.0.5

**Comment by dirkhas** [comment]
This also applies when using the `io.micronaut.mqtt:micronaut-mqttv3` dependency. Builds are broken for clients that use `micronaut-jackson-databind` instead of `micronaut-serde`. Only when excluding the transitive `micronaut-serde-jackson` dependency does it build.

**Comment by timyates** [comment]
Fixed in MQTT 3.1.1 👍
